src/functionBased/components/TodoItem.js

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from 'react';
+import propTypes from 'prop-types';
+import { FaTrash } from 'react-icons/fa';
+import styles from './TodoItem.module.css';
+
+const TodoItem = (props) => {
```

Comment:
[OPTIONAL] ♻
Consider using spread operator for the props object.

Reply:
Please, refer to my first comment:smile: